src/components/FilterPanel/index.tsx

```diff
@@ -0,0 +1,199 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import PlusIcon from '@material-ui/icons/Add';
+import DeleteIcon from '@material-ui/icons/DeleteOutline';
+import IconButton from '@material-ui/core/IconButton';
+import { getTab, getAttributes, getSelectedFilters, getOperators } from '../../reducers/app/selectors';
```

Comment:
Tabs in Arronax correspond to entities. Instead of `getTab` `selectedTab` etc. let's use the word **entity**

`getEntity`, `selectedEntity` etc. Please fix all over the project